patches/tModLoader/Terraria/ModLoader/VanillaDamageClasses.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Terraria.ModLoader
+{
+	public class Melee : DamageClass
+	{
+		public override string ClassName => Lang.tip[2].Value;
+	}
+
+	public class Ranged : DamageClass
+	{
+		public override string ClassName => Lang.tip[3].Value;
+	}
+
+	public class Magic : DamageClass
+	{
+		public override string ClassName => Lang.tip[4].Value;
+	}
+
+	public class Summon : DamageClass
+	{
+		public override string ClassName => Lang.tip[53].Value;
+	}
+}
```

Comment:
No need to have all these classes, just one `VanillaDamageClass` where you pass the localisation key number